manifests/get_from_github.pp

```diff
@@ -1,17 +1,26 @@
-# private
-define collectd::get_from_github(
-  $localfolder,
-  $source
+define collectd::get_from_github (
+  $localfolder = undef,
+  $source      = undef,
+  $ensure      = present,
+  $revision    = 'master',
 ) {
-  if(!defined(Vcsrepo[$localfolder])){
-    collectd::check_and_install_package { $title:
-      package_name => 'git'
-    } ->
+  validate_string($source)
+  validate_string($revision)
```

Comment:
you are right (already exists). 
I will update the rabbitmq once the new options on packaging is done.

and this PR looks good .. merging.